model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/solve_nonhydro.py

```diff
@@ -1335,7 +1335,7 @@ def run_corrector_step(
             gtx.int32(self._config.divdamp_order),
             self._grid.global_properties.mean_cell_area,
             second_order_divdamp_factor,
-            self._config.nudge_max_coeff,
+            self._config.nudge_max_coeff * 5.0,
```

Comment:
We need to fix this in a way that doesn't hurt us in the future...

Reply:
Consider using `max_nudging_coeff` like in diffusion. And apply same pattern as in diffusion.